lexxpluss_apps/src/board_controller.cpp

```diff
@@ -1275,6 +1275,16 @@ class state_controller { // Variables Implemented
             return;
         }
 
+        // eo_option_1 is used as power source for third inductive sensor
+        {
+            gpio_dt_spec gpio_dev = GET_GPIO(eo_option_1);
+            if (!gpio_is_ready_dt(&gpio_dev)) {
+                LOG_ERR("gpio_is_ready_dt Failed\n");
```

Comment:
I think it would be better to log the error as an eo_option_1 error. I think this would be more useful when trying to identify the problem from the log.
eo_option_1のエラーであるとログに残した方が良いと思います。ログから問題を特定しようとしたときに、その方が役にたつと考えます。

Reply:
対応しました。

Reply:
https://github.com/LexxPluss/LexxHard-SensorControlBoard-Firmware/pull/71/commits/baa5c229c2cd29feb1c3eb37678d65b21d0c617e